### ISSUE TITLE
Improve Cue View in multiple-NIC environments

### ISSUE THREAD

**Issue by sparks-alec** [stagehacks/Cue-View#39]
Some initial ideas!

- Indicate what NICs are available
- Indicate what IP ranges are searchable
- Indicate what NIC is being used for each device

**Comment by jwetzell** [comment]
What happens if two NICs overlap how would you be able to present to the user what NIC is being used.

**Comment by sparks-alec** [comment]
I've added a button near the Search and Refresh buttons that opens a Network Information window. This window lists the attached NICs and their search ranges- but you're right, there's no way to know what NIC is attached to each device.
![Screen Shot 2023-01-05 at 2 01 09 AM](https://user-images.githubusercontent.com/919746/210720939-a4355b72-99b7-4523-bf5c-3ceb3c1ce613.png)

**Comment by sparks-alec** [comment]
interesting!! I do want to keep the magic "just show me everything on my show" button that searches everything, but adding narrow searches on this window is a great idea. Probably it needs a new icon then as it transforms from an info-only window to a interactive one.

**Comment by jwetzell** [comment]
Yeah, or maybe have a right-click on the search button give a selection list for individual NICs (left-click would still be the all powerful search)

**Comment by sparks-alec** [comment]
I propose that further development on this be post-1.0!

**Comment by jwetzell** [comment]
I second this proposal. But yeah I think the new information window allows users to see what network interfaces the app is aware of. Only thing I just committed a fix for was I noticed the menu is shown on that window for linux & mac... which to me doesn't make sense so I set it to be removed for those platforms.
